pkg/nfd-master/nfd-master.go

```diff
@@ -1219,3 +1221,19 @@ func (m *nfdMaster) instanceAnnotation(name string) string {
 	}
 	return m.args.Instance + "." + name
 }
+
+func (m *nfdMaster) startNfdControllerApi() error {
```

Comment:
One nit about naming, I'd use name ApiController instead of ControllerApi

```suggestion
func (m *nfdMaster) startNfdApiController() error {
```